cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/curator-oauth-proxy-rb/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: proxy-rolebinding
```

Comment:
This should be named curator-oauth-proxy-rb (to match both the directory name and the convention you're following in other rolebindings).